vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet-styles.js

```diff
@@ -1055,6 +1058,355 @@ export const spreadsheetStyles = css`
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+    padding: 0 var(--lumo-space-xs);
+    --default-font-family: var(--lumo-font-family) !important;
+    --default-font-size: var(--lumo-font-size-s) !important;
+  } 
+
+  :host([theme~="lumo"]) .v-spreadsheet {
+    border: 1px solid var(--lumo-contrast-20pct);
+    font-family: var(--lumo-font-family);
+    font-size: var(--lumo-font-size-m);
+    color: var(--lumo-body-text-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar {
+    background-color: var(--lumo-base-color);
+    border-bottom: 1px solid var(--lumo-contrast-20pct);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .functionfield,
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .addressfield {
+    font-size: var(--lumo-font-size-s);
+    padding-left: var(--lumo-space-xs);
+    color: var(--lumo-body-text-color);
+    background-color: unset;
+    padding: 0px 0px 0px var(--lumo-space-s);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .arrow {
+    font-size: var(--lumo-font-size-xs);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .fixed-left-panel {
+    border-right: 1px solid var(--lumo-contrast-20pct);
+    background-color: white;
+    background-image: linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .adjusting-right-panel {
+    padding-right: var(--lumo-space-xs);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .addressfield {
+    background-color: var(--lumo-base-color);
+    background-image: linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .functionfield {
+    font: unset;
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .formulaoverlay {
+    color: var(--lumo-base-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .functionbar .formulaoverlay span {
+    border-radius: var(--lumo-border-radius-s);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .sheet .cell.selected-cell-highlight {
+    outline: solid var(--lumo-contrast-80pct) 1px;
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .sheet div.custom-editor-cell {
+    padding: var(--lumo-space-xs);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .sheet > input[type='text'] {
+    margin-left: -1px;
+    margin-top: -1px;
+    box-shadow: var(--lumo-box-shadow-s);
+    color: var(--lumo-body-text-color);
+    outline: 1px solid var(--lumo-primary-color) !important;
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .sheet .floater {
+    background-color: var(--lumo-base-color);
+  }
+  :host([theme~="lumo"]) .v-spreadsheet .top-left-pane,
+  :host([theme~="lumo"]) .v-spreadsheet .top-right-pane,
+  :host([theme~="lumo"]) .v-spreadsheet .bottom-left-pane {
+    /*border-right: 1px solid var(--lumo-contrast-50pct);*/
+    border-bottom: 1px solid  var(--lumo-contrast-50pct);
+  }
```

Comment:
This rule creates a tiny dot at the bottom of the column bar :
![image](https://github.com/user-attachments/assets/a98976a8-05ab-4f76-9faa-96c5ff582967)

When it's there, the element has an `inactive` class, so it could be solved by adding a `:not(.inactive)` rule in the selectors.

Also, there's a property commented out, so please delete it.